### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in China

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10341] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
* SF Bay Area: https://github.com/apache/datafusion/discussions/10340
* Europe: https://github.com/apache/datafusion/discussions/10342
* China: https://github.com/apache/datafusion/discussions/10341
* East Coast: https://github.com/apache/datafusion/discussions/10343

**Comment by liukun4515** [comment]
cc @tisonkun @waynexia 
[greptimedb](https://github.com/GreptimeTeam/greptimedb) use the datafusion deeply, I think we can find co-meetup to introduce the datafusion in china

**Reply by tisonkun** [reply]
Cool! This is exactly possible and I'm interested in such a co-meetup.

May I ask your plan on the time, location and scale of the meetup?

**Reply by waynexia** [reply]
Sounds fascinating! I'm very willing to help and attend.

IIRC @yahoNanJing (and others) from ebay also uses datafusion/ballista internally, do you have interest in this?

**Reply by alamb** [reply]
I think @liukun4515  (who also spoke at an ASF conference https://www.youtube.com/watch?v=AJU9rdRNk9I about DataFusion) may also be part of that team / have some ideas

**Reply by waynexia** [reply]
>may also be part of that team

Ahh, sorry for that 😂

Drafted a proposal: https://docs.google.com/document/d/1NLGmmfpI4rIdV3pN2ZxM_Rfo5sevTwnT7hhco4NYVUA/edit

Feel free to edit & comment 🙌

**Reply by liukun4515** [reply]
I will take a look the proposal carefully this weekend

**Reply by caicancai** [reply]
looking forward

**Comment by SteveLauC** [comment]
Excited for this!

**Comment by Xuanwo** [comment]
I'm interested in sharing opendal community's work about Datafusion integration (if this event happen in BEIJING)

**Comment by liurenjie1024** [comment]
Super excited!

**Comment by jackwener** [comment]
Exciting news!

**Comment by tisonkun** [comment]
For anyone in this thread, @waynexia and I are organizing a meetup at Hangzhou, probably on July 21st.

If you're interested in giving a presentation or attending the meetup, feel free to DM us or leave a comment here, so that we can invite you and collaborate together.

Details are updated to https://docs.google.com/document/d/1NLGmmfpI4rIdV3pN2ZxM_Rfo5sevTwnT7hhco4NYVUA

**Reply by tisonkun** [reply]
FYI - @liukun4515 & @chunshao90 has committed to present.

**Reply by jackwener** [reply]
I'm also interested in it.

**Reply by liurenjie1024** [reply]
Interested attending the meetup.

**Reply by r4ntix** [reply]
Wow, I'm interested in attending the meetup in Hangzhou!

**Reply by tisonkun** [reply]
The register link is available here: https://www.huodongxing.com/event/5761971909400?td=1965290734055

Of course, for free :D

**Comment by waynexia** [comment]
It was extremely excited to meet so many fellows in person! 

For those who might be interested, here is the topic list:

- What is the Apache DataFusion and user case in eBay ([slides](https://github.com/user-attachments/files/16327524/ebay_.datafusion.07-21.hangzhou.pdf)) by @liukun4515 
- Build a distributed execution engine in GreptimeDB with Apache DataFusion ([slides](https://github.com/user-attachments/files/16327536/DataFusionMeetUp2024Hangzhou.pdf)) by @waynexia
- Applying DataFusion in Apache HoraeDB, Practices and Challenges ([slides](https://slides.liujiacai.net/datafusion-in-horaedb/)) by @jiacai2050

**Reply by alamb** [reply]
I also found a picture from @jiacai2050  on twitter: https://x.com/liujiacai/status/1815043486078677224

![GTBSouraEAA6VjY](https://github.com/user-attachments/assets/d4e85fef-9dfc-47c6-ba67-dba6a3ad63c3)

**Reply by Xuanwo** [reply]
Coooool!

**Comment by alamb** [comment]
Let's move discussion about the next meetup here: https://github.com/apache/datafusion/discussions/16334
